module/userExtension/helper.js

```diff
@@ -0,0 +1,242 @@
+
+let sunBirdService =
+    require(ROOT_PATH + "/generics/services/sunbird");
+
+/**
+* user related information be here.
+* @method
+* @class  userExtensionHelper
+*/
+module.exports = class userExtensionHelper {
+
+
+    /**
+    * getProfile.
+    * @method
+    * @name  getProfile
+    * @param  {queryObject, token}  - requested queryObject and token.
+    * @returns {json} Response consists of user profile details.
+    */
+    static getProfile(queryObject, token) {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                let profileData = await sunBirdService.getUserProfileInfo(token, queryObject.userId);
+                let roles;
+                profileData = JSON.parse(profileData);
+                if (profileData.responseCode == messageConstants.common.RESPONSE_OK) {
```

Comment:
@deepakr123 , @rakeshSgr - What if the response code is not OK, how are we handling that?